manifests/base/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- github.com/operate-first/continuous-deployment/manifests/base?ref=v1.8.1-1
+- github.com/operate-first/apps/argocd/base?ref=999d2cacb8efe9f06c6018ae0461d0fda3724d6b
```

Comment:
Any chance to replace with a human-readable ref instead of a hash?

Reply:
See discussion why we're using commits for now here: https://github.com/operate-first/continuous-deployment/issues/113#issuecomment-820995779